CRM/Contribute/Form/Contribution.php

```diff
@@ -377,10 +377,6 @@ public function setDefaultValues() {
       $defaults['fee_amount'] = CRM_Utils_Money::format($defaults['fee_amount'], NULL, '%a');
     }
 
-    if (isset($defaults['net_amount'])) {
```

Comment:
Sorry not sure what you mean by 'introducing this bug'? Removing the incorrect calculation here actually fixes net_amount - as it's calculated later on.